platform/core/src/ffi/mod.rs

```diff
@@ -1,15 +1,29 @@
-#[cfg(target_os = "ios")]
+//#[cfg(target_os = "ios")]
```

Comment:
Any reason for this to be commented out?

Reply:
I forgot to uncomment it